cmd/cluster-api-actuator-pkg-tests-ext/main.go

```
@@ -0,0 +1,128 @@
+	. "github.com/onsi/ginkgo/v2"
+
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/klog"
+
+	osconfigv1 "github.com/openshift/api/config/v1"
+	machinev1 "github.com/openshift/api/machine/v1beta1"
+	caov1alpha1 "github.com/openshift/cluster-autoscaler-operator/pkg/apis"
+	awsv1 "sigs.k8s.io/cluster-api-provider-aws/v2/api/v1beta2"
+	azurev1 "sigs.k8s.io/cluster-api-provider-azure/api/v1beta1"
+	gcpv1 "sigs.k8s.io/cluster-api-provider-gcp/api/v1beta1"
+	clusterv1 "sigs.k8s.io/cluster-api/api/core/v1beta1"
```

golangci-lint (2.11.4)

[error] import "sigs.k8s.io/cluster-api/api/core/v1beta1" imported as "clusterv1" but must be "clusterv1beta1" according to config

(importas)